### DISCUSSION TITLE
Themes Linux AppImage

### DISCUSSION THREAD

**Discussion by vcg3rd** [GitSquared/edex-ui#1097] [Questions] [answered]
I take it using different themes isn't possible with an AppImage?  I can't find an option in edex or directions in the F.A.Q or in Discussions here.

**Comment by GitSquared** [comment, marked as answer]
You should check out the [wiki](https://github.com/GitSquared/edex-ui/wiki).

**Reply by vcg3rd** [reply]
Doh!  Of course there'd be a wiki.  Sorry!

**Reply by GitSquared** [reply]
No problem. Enjoy!
